### ISSUE TITLE
Restructure projects page to include "official" ones and clarify the purpose of projects

### ISSUE THREAD

**Issue by WasabiFan** [ev3dev/ev3dev.github.io#269]

**Comment by dlech** [comment]
What are you calling "official" projects?

**Comment by ddemidov** [comment]
http://www.lego.com/en-us/mindstorms/build-a-robot

**Comment by dlech** [comment]
Got it. In that case, I would like to see 2 separate pages, on for user-submitted creations (our current projects page) and another page for official LEGO models.

**Comment by WasabiFan** [comment]
Yeah, I think that's the direction we're going in. You can see some more discussion in #224.

**Comment by WasabiFan** [comment]
What do we want to call these two pages? I could see one being "Community Creations" and the other being "Build a Robot". Thoughts?

**Comment by ddemidov** [comment]
Maybe 'Community showcase' instead of 'creations'?

**Comment by WasabiFan** [comment]
I like the fact that "community showcase" doesn't sound as promotional or artificial as "community creations", but at the same time the meaning of "community showcase" isn't obvious.

**Comment by ddemidov** [comment]
Featured projects?
Community projects?
Robot gallery?
Community robots?
